src/test/java/de/zalando/ep/zalenium/proxy/DockerSeleniumRemoteProxyTest.java

```diff
@@ -557,6 +557,76 @@ public void videoRecordingIsDisabledViaCapability() {
       Assert.assertFalse(proxy.isVideoRecordingEnabled());
     }
 
+    @Test
```

Comment:
Can we add an end to end test as well please?

Reply:
Sure, can you point me to some example?

Reply:
End to End tests are here: https://github.com/zalando/zalenium/blob/master/src/test/java/de/zalando/ep/zalenium/it/ParallelIT.java

Just add one that only runs on Zalenium and uses the added cookie names. Afterwards it should check that the amount of videos matches the amount of tests.